### ISSUE TITLE
"usage" example refers to files no longer present in sciclone-meta repo

### ISSUE THREAD

**Issue by ttriche** [genome/sciclone#20]
It's a bit confusing to discover that the per-tumor VAF files referred to in Usage have been merged into one file in sciclone-meta/tests/data.  So even if one starts out in sciclone-meta/tests, the code in Usage will not run.  It might be a good idea to update this since it documents the known working input format for point mutations and SVs (which is of interest to me as I'd like to replace the latter with GRanges-based ploidy calls).

**Comment by chrisamiller** [comment]
Yeah, I see your point - I'll see about making them match up better soon.  In the meantime, there is some testing code in that meta package that should be useful as well.  Thanks for the feedback!

**Comment by ttriche** [comment]
No worries, I forked it to start patching this.  I shouldn't bitch about
anything I'm not willing to fix.

--t
